src/core/threads.c

```diff
@@ -262,3 +263,13 @@ void MVM_thread_join_foreground(MVMThreadContext *tc) {
         }
     }
 }
+
+MVMint64 MVM_cpucores() {
```

Comment:
right, I did not see MVM_platform_cpu_count. it appears unused in moar, and does not map to an op. but could of course be used by some other libmoar client. The implementation within libuv seems more portable and complete, than the one in MVM_platform_cpu_count.

how about I use MVM_platform_cpu_count for the "cpucores" op, but switch the implementation to the libuv-based one? comments?